packages/e2e/features/ui/components/authenticator/sign-up.feature

```diff
@@ -5,24 +5,26 @@ Feature: Sign Up
   Background:
     Given I'm running the example "ui/components/authenticator/sign-up"
     And I click "Create account"
-    Then I see "Create a new account"
 
-  @next @react @vue @angular @skip
-  Scenario: Sign up with a new username & password
-    When I type a new username
-    And I type a new password
-    And I type a new confirm password
-    And I type the email "test@example.com"
-    And I type the phone number "+15558675309"
+  @angular @next @vue
```

Comment:
`@react` needs to be here because, otherwise, this feature isn't supported on this platform:

> ![Screen Shot 2021-09-10 at 4 20 05 PM](https://user-images.githubusercontent.com/15182/132918971-2bb72612-1e4a-47b3-bff2-ca4f9a49e0fc.png)
> – https://ui.docs.amplify.aws/ui/components/authenticator?platform=react#sign-up


> ![Screen Shot 2021-09-10 at 4 20 14 PM](https://user-images.githubusercontent.com/15182/132918972-6e87a7b3-cd76-42c4-9544-e4ec77989a5a.png)
> – https://ui.docs.amplify.aws/ui/components/authenticator?platform=next#sign-up